store/datastore/users.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/drone/drone/model"
 	"github.com/russross/meddler"
+	"sort"
 )
```

Comment:
no worries :)

appreciate you taking the time to fix the issue and learn Go along the way!